apps/src/p5lab/P5Lab.js

```diff
@@ -571,54 +548,31 @@ export default class P5Lab {
   }
 
   /**
-   * Export the project for web or use within Expo.
-   * @param {Object} expoOpts
+   * Export the project for web.
    */
-  async exportApp(expoOpts) {
+  async exportApp() {
     await this.whenAnimationsAreReady();
     return this.exportAppWithAnimations(
       project.getCurrentName() || 'my-app',
-      getStore().getState().animationList,
-      expoOpts
-    );
-  }
-
-  setAndroidExportProps(props) {
-    // Spread the previous object so changes here will always fail shallow
-    // compare and trigger react prop changes
-    this.generatedProperties.export = {
-      ...this.generatedProperties.export,
-      android: props
-    };
-    project.projectChanged();
-    project.saveIfSourcesChanged();
-    getStore().dispatch(
-      setExportGeneratedProperties(this.generatedProperties.export)
+      getStore().getState().animationList
     );
   }
 
   /**
    * Export the project for web or use within Expo.
```

Comment:
Can we update this comment to remove 'within Expo'?

Reply:
Done, good eye!